src/components/FilterByCentury/CenturyButton.tsx

```diff
@@ -0,0 +1,34 @@
+import classNames from 'classnames';
+import { SearchLink } from '../SearchLink/SearchLink';
+import React from 'react';
+import { useSearchParams } from 'react-router-dom';
+
+type Props = {
+  century: string;
+};
+
+export const CenturyButton: React.FC<Props> = ({ century }) => {
+  const [searchParams] = useSearchParams();
+  let centuries = searchParams.getAll('centuries');
+
+  if (centuries.includes(century)) {
+    centuries = centuries.filter(currentCentury => currentCentury !== century);
+  } else {
+    centuries.push(century);
+  }
+
+  return (
+    <SearchLink
+      key={century}
+      data-cy="century"
+      className={classNames('button', 'mr-1', {
+        'is-info': !centuries.includes(century),
```

Comment:
The logic for setting the button's class uses 'is-info' when the century is not included in the search parameters. This might be counterintuitive as typically, a selected item would have a distinct style. Consider using 'is-info' when the century is included in the search parameters to indicate selection.